include/cuco/detail/__config

```diff
@@ -32,3 +46,7 @@
 #if (CUCO_CUDA_MINIMUM_ARCH >= 700)
 #define CUCO_HAS_INDEPENDENT_THREADS
 #endif
+
+#if defined(__SIZEOF_INT128__) && (CUDART_VERSION) && (CUDART_VERSION >= 11060)
+#define CUCO_HAS_INT128
```

Comment:
```suggestion
#define CUCO_SUPPORTS_INT128
```
`HAS` or `SUPPORTS`?

Reply:
I was following the same naming scheme as in `CUCO_HAS_INDEPENDENT_THREADS`. One could argue that the same applies here, i.e., `CUCO_SUPPORTS_INDEPENDENT_THREADS`.